### PR TITLE
[CI:DOCS] Fix up example description of podman-create.1.md.in

### PR DESCRIPTION
**Pull request by rhatdan**
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: docs/source/markdown/podman-create.1.md.in [hunk 1]
```diff
@@ -387,92 +387,75 @@ Use the **--group-add keep-groups** option to pass the user's supplementary grou
...
-the UIDs and GIDs from the host.
-
+the UIDs and GIDs from the host:
 ```
 $ podman create --uidmap 0:30000:7000 --gidmap 0:30000:7000 fedora echo hello
 ```
 
-### Setting automatic user namespace separated containers
-
+Setting automatic user namespace separated containers:
```
Comment (vrothberg) [inline docs/source/markdown/podman-create.1.md.in:411]:
```suggestion
Setting automatic user-namespace separated containers:
```

File: docs/source/markdown/podman-create.1.md.in [hunk 2]
```diff
@@ -387,92 +387,75 @@ Use the **--group-add keep-groups** option to pass the user's supplementary grou
...
 $ podman create --tz=local alpine date
 $ podman create --tz=Asia/Shanghai alpine date
 $ podman create --tz=US/Eastern alpine date
 ```
 
-### Adding dependency containers
-
-Podman makes sure the first container, container1, is running before the second container (container2) is started.
-
+Podman makes sure the first container, container1, is running before the second container (container2) is started:
```
Comment (vrothberg) [inline docs/source/markdown/podman-create.1.md.in:423]:
```suggestion
Podman makes sure the first container (container1) is running before the second container (container2) is started:
```

**Review by vrothberg — COMMENTED** [review]
Some inconsistencies in using articles

**Review by vrothberg — COMMENTED** [review]
LGTM

File: docs/source/markdown/podman-create.1.md.in [hunk 3]
```diff
@@ -387,92 +387,75 @@ Use the **--group-add keep-groups** option to pass the user's supplementary grou
...
-### Setting automatic user namespace separated containers
-
+Setting automatic user-namespace separated containers:
 ```
 # podman create --userns=auto:size=65536 ubi8-init
 ```
 
-### Configure timezone in a container
-
+Configure timezone in a container:
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-create.1.md.in:416]:
```suggestion
Configure the timezone in a container:
```

File: docs/source/markdown/podman-create.1.md.in [hunk 4]
```diff
@@ -387,92 +387,75 @@ Use the **--group-add keep-groups** option to pass the user's supplementary grou
...
 $ podman create --tz=local alpine date
 $ podman create --tz=Asia/Shanghai alpine date
 $ podman create --tz=US/Eastern alpine date
 ```
 
-### Adding dependency containers
-
-Podman makes sure the first container, container1, is running before the second container (container2) is started.
-
+Podman makes sure the first container (container1) is running before the second container (container2) is started:
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-create.1.md.in:423]:
```suggestion
Ensure the first container (container1) is running before the second container (container2) is started:
```

File: docs/source/markdown/podman-create.1.md.in [hunk 5]
```diff
@@ -387,92 +387,75 @@ Use the **--group-add keep-groups** option to pass the user's supplementary grou
...
-### Exposing shared libraries inside of container as read-only using a glob
-
+Expose shared libraries inside of container as read-only using a glob:
 ```
 $ podman create --mount type=glob,src=/usr/lib64/libnvidia\*,ro -i -t fedora /bin/bash
 ```
 
-### Configure keep supplemental groups for access to volume
-
+Configure container to keep supplemental groups for access to volume:
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-create.1.md.in:443]:
```suggestion
Create a container allowing supplemental groups to have access to the volume:
```

File: docs/source/markdown/podman-create.1.md.in [hunk 6]
```diff
@@ -387,92 +387,75 @@ Use the **--group-add keep-groups** option to pass the user's supplementary grou
...
 ```
 $ podman create -v /var/lib/design:/var/lib/design --group-add keep-groups ubi8
 ```
 
-### Configure execution domain for containers using personality flag
-
+Configure execution domain for containers using personality flag:
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-create.1.md.in:448]:
```suggestion
Configure execution domain for containers using the personality option:
```

**Pull request merged**